src/App.tsx

```diff
@@ -206,6 +206,17 @@ function App() {
                 </small>
               </li>
 
+              <li>
+                For convenience, you can store trust anchors and safebags in QR code and scan them for quick fill-in. On
+                Linux-based OS's, the following command converts text to QR code (using trust anchor as example):
```

Comment:
I know it may look obvious, but still better to say something like "you may need a display for it"?

Reply:
> I know it may look obvious, but still better to say something like "you may need a display for it"?

Added